libexec/bootstrap-scripts/main-deffile.sh

```diff
@@ -53,9 +53,23 @@ if [ -n "${BOOTSTRAP:-}" -a -z "${SINGULARITY_BUILDNOBASE:-}" ]; then
     fi
 fi
 
+# take a snapshot of the environment for later comparison
+if [ ${BOOTSTRAP} = "localimage" ]; then
+    export SINGULARITY_STARTING_ENVIRONMENT=$(eval_abort env -i ${SINGULARITY_libexecdir}/singularity/helpers/record-env.sh ${SINGULARITY_ROOTFS})
+    export SINGULARITY_STARTING_ENVSHA1=$(eval_abort sha1sum ${SINGULARITY_ROOTFS}/.singularity.d/env/*.sh | sha1sum)
+fi 
```

Comment:
this is cool! So the first command is the environment (keys and values) and the second is a hash for it? Can you show me what the two variables look like?